docs/administration-guide/gaffer-config/config.md

```diff
@@ -1,8 +1,11 @@
 ## How to configure a Gaffer instance
 
-First you must set up the files and directories you will need for the instance. As it stands there
-are a couple of different ways to run a Gaffer project this example will use a logical structure
-that suites a stand alone deployment e.g. docker consisting of the following file structure:
+To configure a Gaffer instance you must first set up the files and directories
+needed for your instance. 
+
+As it stands there are a couple of different ways to run a Gaffer project this
```

Comment:
I'd just remove this bit and the 'example project structure' as its just repeating whats in the example deployment. This could just be a general intro like:
"There are various configuration files for Gaffer, this page gives an overview of the commonly used files and how to tweak them for your project needs.